cmd/release-controller/types.go

```diff
@@ -304,6 +304,9 @@ const (
 	// releaseAnnotationFromImageStream specifies the imagestream
 	// a release was promoted from. It has the format <namespace>/<imagestream name>
 	releaseAnnotationFromImageStream = "release.openshift.io/from-image-stream"
+
+	// Art specified builder tag to provide the correct "cli" image to run for multi-arch support
+	releaseAnnotationBuilder = "release.openshift.io/builder"
```

Comment:
Why isn't this art of the release config?  We already have an override image?

Reply:
Specifically https://github.com/openshift/release-controller/pull/123/files#diff-f08ed12693cad4475338e0ae3f8eb292R72

Reply:
ugh did not know this existed already, yes this is exactly what we need, we can close this PR than and just update the config for the relevant release image streams.